packages/styleguide/.storybook/components/Page/index.tsx

```diff
@@ -35,41 +38,88 @@ const HeaderRow = styled.div`
 
 const HeaderCol = styled.div``;
 
-interface Parameters {
-  component?: React.Component | string;
-  subcomponents?: Record<string, React.Component>;
-  pageTitle?: string;
+export interface GamutParameters extends Parameters {
+  subtitle?: string;
   status?: 'stable' | 'volatile' | 'deprecated';
   figmaId?: string;
   source?: string;
 }
 
+const BreadCrumbs: React.FC<{ path: string[] }> = ({ path }) => {
+  const { storyStore } = useContext(DocsContext);
+  if (path[0] === 'Gamut') return null;
+
+  let links = [];
+
+  path.forEach((kind, i) => {
+    const prevKind = i ? `${links[i - 1].kind}/` : '';
+
+    const nextLink = {
+      text: kind,
+      kind: `${prevKind}${kind}`,
+    };
+
+    links.push(nextLink);
+  });
+
+  links = [{ text: 'Gamut', kind: 'Gamut' }, ...links];
```

Comment:
⛳ ✨  _(I haven't checked if this works exactly, but generally something like this?)_

```suggestion
  const links = [{ text: 'Gamut', kind: 'Gamut' }];

  path.forEach((kind, i) => {
    const prevKind = i ? `${links[i].kind}/` : '';

    const nextLink = {
      text: kind,
      kind: `${prevKind}${kind}`,
    };

    links.push(nextLink);
  });
```